lib/client-assets.php

```diff
@@ -1357,6 +1357,8 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'post' => $post->ID,
 	) );
 	wp_enqueue_editor();
+	wp_playlist_scripts( 'audio' );
```

Comment:
Shouldn't `wp.playlist` to be listed as a dependency of `wp-block-library`? It's referenced it in the code so I would assume that it needs to be.